server/executor/trigger/http.go

```diff
@@ -73,7 +73,7 @@ func (te *httpTriggerer) Trigger(ctx context.Context, test model.Test, opts *Tri
 	client := httpClient()
 
 	ctx = trace.ContextWithSpanContext(ctx, newSpanContext(ctx))
-	ctx, cncl := context.WithTimeout(ctx, 5*time.Second)
+	ctx, cncl := context.WithTimeout(ctx, 30*time.Second)
```

Comment:
Some services take more than 5 seconds, I increased this in the meantime but we should consider having this part of config or something like that

Reply:
Created ticket to add this later to the trigger: https://github.com/kubeshop/tracetest/issues/2079

Reply:
Thanks!